pysparkle/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "pysparkle"
+version = "0.1.0"
+description = "PySpark data transformer"
+authors = ["Paulina Motyl <paulina.motyl@ensono.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
This has been tested both on 3.9 and 3.10. I'm allowing 3.9 because the latest Databricks LTS runtime has Python 3.9.5.